trail/gallery/templates/images.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block gallery-nav-state%}active{% endblock %}
+
+{% block content %}
+<!-- Featured Post -->
+<article class="post featured">
+    <header class="major">
+		    <span class="date">{{ image_data.datetime_begin}}</span>
+    </header>
+    <!--This will probably need to be changed-->
```

Comment:
When? Also, if you add `TODO: Change when transition to .... blabla` then one can easily perform a text base search for all `TODO`s, `INFO`, `WARNINGS`, etc...